search-endpoint.md

```diff
@@ -169,6 +165,17 @@ The returned JSON response will be like:
     "facets" : [
       {
         "name" : "author",
+        "facetType": "text",
+        "hasMore": true,
+        "facetLimit": 5,
```

Comment:
this is a duplication of the page.size information

Reply:
This is [the discovery configuration setting](https://github.com/DSpace/DSpace/blob/master/dspace/config/spring/api/discovery.xml#L375) and can differ from the page size (e.g. when the user supplies a custom page size).

Reply:
ok so we need to document the meaning of this facetLimit... is it the default size or the maximum size that a page can have? In the current UI it is used with both meaning as the facet page size is not alterable

Reply:
It will be the default page size. So strictly speaking it is not required by the UI (as it should just look at the page size). But it doesn't break anything if we expose the facet configuration?

Reply:
I will prefer to be conservative and don't expose something that is not strictly needed avoiding the risk that someone will start to use it in a way that we don't expect.